tests/functional/curate/cram/titlecase.t

```diff
@@ -0,0 +1,46 @@
+Setup
+
+  $ pushd "$TESTDIR" > /dev/null
+  $ export AUGUR="${AUGUR:-../../../../bin/augur}"
+
+Test output with lowercase and already captialized strings.
+
+  $ echo '{"record": 1, "authors": "john smith", "author2": "Jane Doe"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "authors" "author2"
+  {"record": 1, "authors": "John Smith", "author2": "Jane Doe"}
+
+Test output with lowercase articles.
+
+  $ echo '{"record": 2, "title": "the night OF THE LIVING DEAD", "location": "BRAINE-LE-COMTE, FRANCE"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "title" "location" --articles "a" "and" "of" "the" "le"
+  {"record": 2, "title": "The Night of the Living Dead", "location": "Braine-le-Comte, France"}
+
+Test output with abbreviations
+
+  $ echo '{"record": 3, "city": "Washington DC, USA"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "city" --abbreviations "USA" "DC"
+  {"record": 3, "city": "Washington DC, USA"}
+
+Test output with unicode characters
+
+  $ echo '{"record": 4, "location": "Auvergne-Rhône-Alpes" }' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "location"
+  {"record": 4, "location": "Auvergne-Rh\u00f4ne-Alpes"}
+
+Test output with numbers
+
+  $ echo '{"record": 5, "title": "2021 SARS-CoV"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "title" --abbreviations "SARS"
+  {"record": 5, "title": "2021 SARS-Cov"}
+
```

Comment:
_non-blocking_

I might also add a test here for trying to titlecase a field that is just a numeric value (not part of a string). 
With the default `--failure-reporting`, the numeric value field would raise an error.

Reply:
Neat, added as part of https://github.com/nextstrain/augur/pull/1197/commits/3c20dcb964ddfa5d40047eff92e223aab4a2a766